src/Illuminate/Support/UrlQueryParameters.php

```diff
@@ -71,4 +72,20 @@ public function toArray(): array
     {
         return $this->parameters;
     }
+
+    /**
+     * Convert the query parameters to a string.
+     *
+     * @return string
+     */
+    public function __toString(): string
+    {
+        $parameters = [];
+
+        foreach ($this->parameters as $key => $value) {
+            $parameters[] = $key.'='.$value;
+        }
+
+        return implode('&', $parameters);
```

Comment:
This could be:
```suggestion
        return implode('&', array_map(fn($key, $value) => $key.'='.$value, array_keys($this->parameters), array_values($this->parameters)));
```

Reply:
[http_build_query](https://www.php.net/manual/en/function.http-build-query.php)?

Reply:
Yeah, sometimes, one really is unable to see the wood for the trees 🤦🏻‍♂️😅

Reply:
Done, thanks @hafezdivandari! 👍